0.12 has a "failed to parse generated bindings" error

multiple errors like this
```
       Using DefaultFeature("pg13") and `pg_config` from /home/ubuntu/.pgrx/13.11/pgrx-install/bin/pg_config
CARGO_TARGET_DIR="/home/ubuntu/ris_avails_r_ext/target" PGRX_ALL_FEATURES="false" PGRX_BUILD_PROFILE="dev" PGRX_FEATURES="pg13 pg_test" PGRX_NO_DEFAULT_FEATURES="true" PGRX_NO_SCHEMA="false" "/home/ubuntu/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/cargo" "test" "--features" "pg13 pg_test" "--no-default-features"
   Compiling pgrx-pg-sys v0.12.0
error: failed to run custom build command for `pgrx-pg-sys v0.12.0`

Caused by:
  process didn't exit successfully: `/home/ubuntu/ris_avails_r_ext/target/debug/build/pgrx-pg-sys-d09c9452236bbdcb/build-script-bindgen` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=PGRX_BUILD_VERBOSE
  cargo:rerun-if-env-changed=PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE
  cargo:rerun-if-env-changed=PGRX_PG_CONFIG_PATH
  cargo:rerun-if-env-changed=PGRX_PG_CONFIG_AS_ENV
  cargo:rerun-if-env-changed=LLVM_CONFIG_PATH
  cargo:rerun-if-env-changed=LIBCLANG_PATH
  cargo:rerun-if-env-changed=LIBCLANG_STATIC_PATH
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE
  cargo:rerun-if-changed=include
  cargo:rerun-if-changed=cshim
  cargo:rerun-if-changed=/home/ubuntu/.pgrx/config.toml
  cargo:rerun-if-env-changed=PGRX_TARGET_INFO_PATH_PG13_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PGRX_TARGET_INFO_PATH_PG13
  cargo:rerun-if-env-changed=PGRX_BINDGEN_NO_DETECT_INCLUDES_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PGRX_BINDGEN_NO_DETECT_INCLUDES
  cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER_PG13_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER_PG13
  cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER
  cargo:rerun-if-env-changed=PGRX_PG_SYS_EXTRA_OUTPUT_PATH

  --- stderr
  build_paths=BuildPaths { manifest_dir: "/home/ubuntu/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.12.0", out_dir: "/home/ubuntu/ris_avails_r_ext/target/debug/build/pgrx-pg-sys-f886f3e2945f3d25/out", src_dir: "/home/ubuntu/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.12.0/src/include", shim_src: "/home/ubuntu/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.12.0/cshim", shim_dst: "/home/ubuntu/ris_avails_r_ext/target/debug/build/pgrx-pg-sys-f886f3e2945f3d25/out/cshim" }
  Generating bindings for pg13
  pg_config --configure CLANG = None
  Bindgen found clang version 10.0.0-4ubuntu1 
  found libclang at /usr/lib/llvm-10/lib/libclang-10.0.0.so
  Found include dirs ["/usr/lib/llvm-10/lib/clang/10.0.0/include"]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/port.h:185:70: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/port.h:187:55: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/port.h:189:58: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/port.h:191:43: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:157:40: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:158:49: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:161:30: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:161:56: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:163:43: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:164:52: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:166:47: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:170:31: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:170:57: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:173:30: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:173:56: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:175:41: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:189:48: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:221:54: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/elog.h:437:47: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/palloc.h:155:44: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/utils/palloc.h:156:80: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/lib/stringinfo.h:96:67: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  clang diag: /home/ubuntu/.pgrx/13.11/pgrx-install/include/postgresql/server/lib/stringinfo.h:107:78: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
  Error: bindgen failed for pg13

  Caused by:
     0: failed to parse generated bindings
     1: expected identifier or `_`

  Location:
      /home/ubuntu/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-bindgen-0.12.0/src/build.rs:743:40
```

--- a/pgrx-bindgen/src/build.rs
+++ b/pgrx-bindgen/src/build.rs
@@ -114,7 +114,7 @@ impl bindgen::callbacks::ParseCallbacks for BindingOverride {
         enum_name.inspect(|name| match name.strip_prefix("enum").unwrap_or(name).trim() {
             // specifically overridden enum
             "NodeTag" => return,
-            name if name.contains("unnamed at") => return,
+            name if name.contains("unnamed at") || name.contains("anonymous at") => return,
             // to prevent problems with BuiltinOid
             _ if variant_name.contains("OID") => return,
             name => self

